packages/contracts-bedrock/contracts/dispute/interfaces/IDisputeGameFactory.sol

```diff
@@ -5,105 +5,87 @@ import "../../libraries/DisputeTypes.sol";
+    /// @param extraData Any extra data that should be provided to the created dispute game.
+    /// @return _proxy The clone of the `DisputeGame` created with the given parameters.
+    ///         Returns `address(0)` if nonexistent.
+    /// @return _timestamp The timestamp of the creation of the dispute game.
     function games(
         GameType gameType,
         Claim rootClaim,
         bytes calldata extraData
     ) external view returns (IDisputeGame _proxy, uint256 _timestamp);
 
-    /**
-     * @notice `gameAtIndex` returns the dispute game contract address and its creation timestamp
-     *          at the given index. Each created dispute game increments the underlying index.
-     * @param _index The index of the dispute game.
-     * @return _proxy The clone of the `DisputeGame` created with the given parameters.
-     *         Returns `address(0)` if nonexistent.
-     * @return _timestamp The timestamp of the creation of the dispute game.
-     */
+    /// @notice `gameAtIndex` returns the dispute game contract address and its creation timestamp
+    ///          at the given index. Each created dispute game increments the underlying index.
+    /// @param _index The index of the dispute game.
+    /// @return _proxy The clone of the `DisputeGame` created with the given parameters.
+    ///         Returns `address(0)` if nonexistent.
+    /// @return _timestamp The timestamp of the creation of the dispute game.
     function gameAtIndex(uint256 _index)
         external
         view
         returns (IDisputeGame _proxy, uint256 _timestamp);
 
-    /**
-     * @notice `gameImpls` is a mapping that maps `GameType`s to their respective
-     *         `IDisputeGame` implementations.
-     * @param gameType The type of the dispute game.
-     * @return _impl The address of the implementation of the game type.
-     *         Will be cloned on creation of a new dispute game with the given `gameType`.
-     */
+    /// @notice `gameImpls` is a mapping that maps `GameType`s to their respective
+    ///         `IDisputeGame` implementations.
+    /// @param gameType The type of the dispute game.
+    /// @return _impl The address of the implementation of the game type.
+    ///         Will be cloned on creation of a new dispute game with the given `gameType`.
     function gameImpls(GameType gameType) external view returns (IDisputeGame _impl);
 
-    /**
-     * @notice Creates a new DisputeGame proxy contract.
-     * @param gameType The type of the DisputeGame - used to decide the proxy implementation
-     * @param rootClaim The root claim of the DisputeGame.
-     * @param extraData Any extra data that should be provided to the created dispute game.
-     * @return proxy The address of the created DisputeGame proxy.
-     */
+    /// @notice Creates a new DisputeGame proxy contract.
+    /// @param gameType The type of the DisputeGame - used to decide the proxy implementation
```

Comment:
I don't think we've strictly followed, but would be nice to fully punctuate comments
```suggestion
    /// @param gameType The type of the DisputeGame - used to decide the proxy implementation.
```